src/include/miopen/norm/problem_description.hpp

```diff
@@ -23,14 +23,12 @@
  * SOFTWARE.
  *
  *******************************************************************************/
-
 #pragma once
 
-#include <miopen/problem_description_base.hpp>
+#include <cassert>
 #include <miopen/activ.hpp>
+#include <miopen/problem_description_base.hpp>
 #include <miopen/tensor.hpp>
-
-#include <cassert>
```

Comment:
In addition to https://github.com/ROCm/MIOpen/pull/2583/files?diff=split&w=1#r1428594833: Please do not delete empty line. Ditto everywhere.

Please see https://github.com/ROCm/MIOpen/discussions/2638#discussion-5997688 for more info.

Reply:
This does not comply the 2nd bullet from https://github.com/ROCm/MIOpen/discussions/2638#discussion-5997688 yet, but it's acceptable.

[Recommendation] Use empty line to split a list of include files into "sections".